weblate/templates/trans/suggestion_list.html

```diff
@@ -12,6 +12,4 @@
 
 {% include "snippets/suggestions.html" with suggestions=object_list multi_unit=1 %}
 
-{% include "paginator.html" %}
```

Comment:
I think the paginator should stay here. Again, the results are still paginated here, you've removed only the UI.